serverless/pages/search-your-data-semantic-search.asciidoc

```diff
@@ -6,140 +6,48 @@
-
-include::../partials/generate-embeddings-widget.asciidoc[]
-
-Now it is time to perform semantic search!
 
 [discrete]
-[[semantic-search-search]]
-== Search the data
-
-Depending on the type of model you have deployed, you can query sparse vectors
-with a sparse vector query, or dense vectors with a kNN search.
+[[elasticsearch-reference-semantic-search-elser]]
+== Semantic search with ELSER
 
-include::../partials/search-widget.asciidoc[]
+Elastic Learned Sparse EncodeR - or ELSER - is an NLP model trained by Elastic
+that enables you to perform semantic search by using sparse vector
+representation. Instead of literal matching on search terms, semantic search
+retrieves results based on the intent and the contextual meaning of a search
+query.
 
-[discrete]
-[[semantic-search-hybrid-search]]
-== Beyond semantic search with hybrid search
-
-In some situations, lexical search may perform better than semantic search. For
-example, when searching for single words or IDs, like product numbers.
-
-Combining semantic and lexical search into one hybrid search request using
-{ref}/rrf.html[reciprocal rank fusion] provides the best of both worlds. Not only that,
-but hybrid search using reciprocal rank fusion {blog-ref}improving-information-retrieval-elastic-stack-hybrid[has been shown to perform better
-in general].
-
-include::../partials/hybrid-search-widget.asciidoc[]
-
-[discrete]
-[[semantic-search-read-more]]
-== Read more
-
-* Tutorials:
-+
-** <<elasticsearch-reference-semantic-search-elser,Semantic search with ELSER>>
-** {ml-docs}/ml-nlp-text-emb-vector-search-example.html[Semantic search with the msmarco-MiniLM-L-12-v3 sentence-transformer model]
-* Blogs:
-+
-** {blog-ref}may-2023-launch-sparse-encoder-ai-model[Introducing Elastic Learned Sparse Encoder: Elastic's AI model for semantic search]
-** {blog-ref}lexical-ai-powered-search-elastic-vector-database[How to get the best of lexical and AI-powered search with Elastic's vector database]
-** Information retrieval blog series:
-+
-*** {blog-ref}improving-information-retrieval-elastic-stack-search-relevance[Part 1: Steps to improve search relevance]
-*** {blog-ref}improving-information-retrieval-elastic-stack-benchmarking-passage-retrieval[Part 2: Benchmarking passage retrieval]
-*** {blog-ref}may-2023-launch-information-retrieval-elasticsearch-ai-model[Part 3: Introducing Elastic Learned Sparse Encoder, our new retrieval model]
-*** {blog-ref}improving-information-retrieval-elastic-stack-hybrid[Part 4: Hybrid retrieval]
-* Interactive examples:
-+
-** The https://github.com/elastic/elasticsearch-labs[`elasticsearch-labs`] repo contains a number of interactive semantic search examples in the form of executable Python notebooks, using the {es} Python client.
-
-// The include that was here is another page
+Learn how to implement semantic search with ELSER in the https://www.elastic.co/guide/en/elasticsearch/reference/current/semantic-search-elser.html[Elasticsearch docs →].
```

Comment:
```suggestion
Learn how to implement semantic search with ELSER in the {ref}/semantic-search-elser.html[Elasticsearch docs →].
```

don't forget to update all instances of absolute URLs to use attributes, it's a good thing to have on a checklist when finishing up a PR :)